src/opm/output/eclipse/AggregateMSWData.cpp

```diff
@@ -1007,6 +1007,24 @@ namespace {
             void traverseStructure();
 
         private:
+            struct KickOffPoint
+            {
+                int segment;
+                int outlet;
+                int branch;
+
+                bool operator<(const KickOffPoint& that) const
+                {
+                    // We use '>' here to have low outlet segment IDs move
+                    // to the front of the priority queue.  If multiple
+                    // branches kick off from the same outlet segment, we
+                    // sort these by their branch IDs whence lower-ID
+                    // branches are created before higher-ID branches.
+                    return std::tie(this->outlet, this->branch)
+                        >  std::tie(that .outlet, that .branch);
+                }
```

Comment:
This does not seem very expressive. If I am not mistaking then this is an operator< that actually performs the operation of operator>. When this is used people will be quite surprised.

Wouldn't is be better to call it `operator>` and e.g. use `std::greater<KickOffPoint>` as an additional template parameter for `std::priority_queue`?

Reply:
I guess this is not worth the time to discuss further. It is  a private class.

Reply:
> I guess this is not worth the time to discuss further. It is a private class.

Sure, but I appreciate the comment.  I've updated the PR accordingly.